Makefile

```diff
@@ -98,4 +98,7 @@ test: requires_nix_shell
 build: requires_nix_shell
 	cabal build -j$(THREADS)
 
-ci: format_check lint build bench_check test haddock
+ci:
+	@ [[ "$$(uname -sm)" == "Linux x86_64" ]] \
+	  && (nix build .#check.x86_64-linux) \
+	  || (echo "CI only builds on Linux x86_64. Your system is $$(uname -sm). If you want to build it for your system, use 'nix build .#check.<your system>' instead." && false)
```

Comment:
Kinda wish that there's a way to generate the correct arch string....

Reply:
```
❯ nix repl '<nixpkgs>'
warning: future versions of Nix will require using `--file` to load a file
Welcome to Nix 2.10.3. Type :? for help.

Loading installable ''...
Added 16770 variables.
nix-repl> system
"aarch64-darwin"

nix-repl>
```
Hmmm

Reply:
Just for fun:
```bash
nix eval -f '<nixpkgs>' system | cut -c 2- | rev | cut -c 2- | rev
```

Reply:
Hah, I suppose this works. I hadn't thought of doing it this way. I'll do it!

Reply:
You can perhaps use `seq` and make it look nicer :D

Reply:
Actually, we don't need anything!

Reply:
lol I see.